README.md

```diff
@@ -19,6 +19,77 @@ Run Journey Tests: see [here](https://github.com/hmrc/tax-enrolment-assignment-j
 
 Run Performance Tests see [here](https://github.com/hmrc/tax-enrolment-assignment-performance-tests)
 
+## Test Only Endpoints
+There are 3 test only endpoints that can be used to add test data to use the service and all downstream APIs that the service relies upon such as enrolment store, BAS stubs, one-login stubs etc. And one to delete from all the same APIs. 2 are UI based and the others are POST routes
+
+### GET /protect-tax-info/test-only/select-user
+The select-user endpoint presents a page with 22 preset user details, with a range of scenarios for both Government Gateway and One Login users, these include:
+- Single Account with no enrolments
+- Single Account with IR-SA enrolment
+- Multiple Accounts with no enrolments
+- Multiple Account, each with varying enrolments between none, HMRC-PT and IR-SA
+
+After selecting and succesfully submitting a user, you will be taken to a success page that will let you easily log in to the service with any of the accounts added.
+
+### GET /protect-tax-info/test-only/insert-user
+The insert-user end-point provides a more open UI solution to enter data, here you can input data in the text area in the following format, which is an example of a user with one account with an IR-SA enrolment:
+```json
+[
```

Comment:
Is this JSON format documented anywhere? IS there a confluence page or another github repo that describes what is meant by "factorType" of "totp" for example?